frontend/src/modules/signup/stakeholderSchema.js

```diff
@@ -118,35 +79,15 @@ export const schema = {
         id: "tabs",
         value: ["S2"],
       },
-      required: [
-        "orgName",
-        "orgRepresentative",
-        "orgDescription",
-        "orgUrl",
-        "orgLogo",
-      ],
+      required: ["orgName"],
       properties: {
         orgName: {
-          title: "What is the name of your organisation?",
+          title: "Search Your Entity",
           type: "string",
```

Comment:
This entity field will be a selected dropdown or a string field?

Reply:
WIP for the next PR